scenarios/tracking/01_training_introduction.ipynb

```diff
@@ -13,19 +13,36 @@
    "cell_type": "markdown",
```

Comment:
<p>How about:</p><p><code>TrackingLearner</code> defaults to FairMOT's baseline tracking model, which can be downloaded [here] &lt;link to gdrive&gt;. This has been pretrained on pedestrian data, such as those found in the MOT Challenge datasets. In order to track the fridge objects in our use-case, we must finetune the model on our own dataset.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>